<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsuranceDealEdit</class>
 <widget class="QDialog" name="InsuranceDealEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание страховой сделки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Клиент</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="clientEdit">
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Тип страховки</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="insTypeEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Действительна с</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="createdAtEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Действительна до</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="validToEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Стоимость</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="costEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>insTypeEdit</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>InsuranceDealEdit</receiver>
   <slot>calculateCost()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createdAtEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>InsuranceDealEdit</receiver>
   <slot>calculateCost()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>validToEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>InsuranceDealEdit</receiver>
   <slot>calculateCost()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InsuranceDealEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calculateCost()</slot>
  <slot>setInsuranceType(int)</slot>
  <slot>setClient(int)</slot>
 </slots>
</ui>
